core/src/test/scala-2/sttp/tapir/generic/LegacySchemaGenericAutoTest.scala

```diff
@@ -122,4 +123,8 @@ class LegacySchemaGenericAutoTest extends AsyncFlatSpec with Matchers {
     removeValidators(implicitly[Schema[JList]]) shouldBe expectedJSchema
   }
 
+  it should "derive schema for enumeration" in {
+    implicitly[Schema[Countries.Country]] shouldBe Schema(SString())
```

Comment:
I'd expect to have a validator here with the possible values

Reply:
Yes, I trying to work this out based on our `ValidatorEnumerationMacro`. I'am trying to extend it to handle `Enumeration` as well.

Reply:
tried `SchemaAnnotations` but it didn't discovered any annotations when I put them on enum `Value`

Reply:
ah, probably another Scala macros quirk ;) maybe it's on the owner symbol as you're probably passing the type which is a child